docker-compose.yml

```diff
@@ -24,31 +24,54 @@ services:
       GOOGLE_CLIENT_SECRET: ${GOOGLE_CLIENT_SECRET}
       GOOGLE_CALLBACK_URL: ${GOOGLE_CALLBACK_URL}
       COOKIES_REFRESH_MS: ${COOKIES_REFRESH_MS}
-      CORS_ENABLED: ${COOKIES_REFRESH_MS}
+      CORS_ENABLED: ${CORS_ENABLED}
       CORS_ALLOWED_ORIGINS: ${CORS_ALLOWED_ORIGINS}
-      CORS_ALLOWED_METHODS: ${COOKIES_REFRESH_MS}
+      CORS_ALLOWED_METHODS: ${CORS_ALLOWED_METHODS}
     depends_on:
       - db
+      - minio
     networks:
-      - app-network
+      app-network:
```

Comment:
Why it was changed?

Reply:
About environment variables - it's correct changes. We fixed my fault.